code/modules/mining/mine_items.dm

```diff
@@ -192,3 +192,56 @@
 	icon_opened = "miningcaropen"
 	icon_closed = "miningcar"
 
+/*********************Mob Capsule*************************/
+
+/obj/item/device/mobcapsule
+	name = "lazarus capsule"
+	desc = "It allows you to store and deploy lazarus-injected creatures easier."
+	icon = 'icons/obj/mobcap.dmi'
+	icon_state = "mobcap0"
+	w_class = 1.0
+	throw_range = 20
+	var/mob/living/captured = null
+	var/colorindex = 0
+
+/obj/item/device/mobcapsule/attack(var/atom/A, mob/user, prox_flag)
+	if(!istype(A, /mob/living/simple_animal))
+		return ..()
+	capture(A, user)
+	return 1
+
+/obj/item/device/mobcapsule/proc/capture(var/mob/target, var/mob/U as mob)
+	var/mob/living/simple_animal/T = target
+	if(captured)
+		U << "<span class='notice'>Capture failed!</span>: The capsule already has a mob registered to it!"
+		return
+	else
+		if(istype(T) && "neutral" in T.faction)
```

Comment:
The "if(istype(T)" line right there was made because we originally tried binding it to the friend var that malfunctioning lazarus injectors have do. We didn't know that was bound to the malfunctioning lazarus injector though, a difference between /vg/ and Paracode, I guess.